### PR TITLE
Added winID to fix second windows in OGRE2.2

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>

# 🦟 Bug fix

## Summary

Fix https://github.com/ignitionrobotics/ign-gui/issues/291

Requires a PR in ign-rendering6

## Checklist
- [ ] Signed all commits for DCO
- [ ] Added tests
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge**

### CODE REVIEW

File: src/plugins/minimal_scene/MinimalScene.cc
Comment (chapulina) [inline src/plugins/minimal_scene/MinimalScene.cc:471]: I know `Scene3D` is deprecated, but how about doing the same there? For users who are transitioning from one to the other. The same for `GzScene3D`

**Comment by iche033** [comment]
hmm with this change and ignitionrobotics/ign-rendering#436, I get an empty render window and then a crash on render.

I'm on Ubuntu bionic with Qt 5.9.5

**Comment by ahcorde** [comment]
> hmm with this change and [ignitionrobotics/ign-rendering#436](https://github.com/ignitionrobotics/ign-rendering/pull/436), I get an empty render window and then a crash on render.
> 
> I'm on Ubuntu bionic with Qt 5.9.5

you probably used a world with the Gazebo Scene3D here it's the fix https://github.com/ignitionrobotics/ign-gazebo/pull/1063

File: src/plugins/scene3d/Scene3D.cc
Comment (chapulina) [inline src/plugins/scene3d/Scene3D.cc:1122]:
```suggestion
```

**Comment by matosinho** [comment]
Issue fixed here for me with this branch.

**Review by chapulina — APPROVED** [review]
Works for me :+1: We just need to wait for the `ign-rendering` release before merging.

**Comment by chapulina** [comment]
Actually there's no harm in merging this before an `ign-rendering` release, so I'll just merge it to check it off

**Pull request merged**
